src/olympia/addons/views.py

```diff
@@ -1239,3 +1247,34 @@ def filter_queryset(self, qs):
     def paginate_queryset(self, queryset):
         # We don't need pagination for the fixed number of results.
         return queryset
+
+
+class AddonBrowserMappingView(ListAPIView):
+    authentication_classes = []
+    serializer_class = AddonBrowserMappingSerializer
+    pagination_class = LargePageNumberPagination
+
+    def get(self, request, format=None):
+        browser = self.request.query_params.get('browser', None)
+        if browser not in list(BROWSERS.values()):
+            raise exceptions.ParseError('Invalid browser parameter')
+
+        return super().get(request, format)
+
+    def get_queryset(self):
+        browser = next(
+            (
+                key
+                for key in BROWSERS
+                if BROWSERS.get(key) == self.request.query_params.get('browser')
+            ),
+            None,
+        )
+        return AddonBrowserMapping.objects.filter(
+            browser=browser, addon__status__in=amo.APPROVED_STATUSES
+        ).annotate(addon__guid=F('addon__guid'))
```

Comment:
Add a comment mentioning this annotation is used in the serializer and allows us to avoid useless SQL queries since we only care about the `guid`